.github/workflows/server-test-template.yml

```diff
@@ -59,7 +59,7 @@ jobs:
         env:
           BUILD_IMAGE: mattermostdevelopment/mattermost-build-server:${{ steps.go.outputs.GO_VERSION }}
         run: |
-          if [[ ${{ github.ref_name }} == 'master' ]]; then
+          if [[ ${{ github.ref_name }} == 'master' && ${{ inputs.fullyparallel }} != true ]]; then
```

Comment:
Apologies if I misunderstand, but I thought this block was to turn on `-race` for `master`, which we don't run anywhere else (because it's slow). Is `fullyparallel` always going to be `false` for `master`?

Reply:
(Thanks for the PR review feedback! Until I have confidence that all PR submitters consistently re-request reviews vs. just comment in reply, I'm probably going to continue being explicit. Plus, I do agree that it's what the reviewer writes that mattersmost, and not the label used.)

---

Your explanation has brought clarity -- thank you! I wonder if a future improvement might be to have an `allowRace` parameter that defaults to `true`, but is set `false` for the fully parallel scenario?

Reply:
> (Thanks for the PR review feedback! Until I have confidence that all PR submitters consistently re-request reviews vs. just comment in reply, I'm probably going to continue being explicit. Plus, I do agree that it's what the reviewer writes that mattersmost, and not the label used.)

:+1: 

> Your explanation has brought clarity -- thank you! I wonder if a future improvement might be to have an `allowRace` parameter that defaults to `true`, but is set `false` for the fully parallel scenario?

Yes, I think that would be clearer.

Reply:
😂 reviewing the review